activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -1070,7 +1072,7 @@ def reverse_sql_order(order_query)
             end
             o.split(",").map! do |s|
               s.strip!
-              s.gsub!(/\sasc\Z/i, " DESC") || s.gsub!(/\sdesc\Z/i, " ASC") || s.concat(" DESC")
+              s.gsub!(/\sasc\Z/i, " DESC") || s.gsub!(/\sdesc\Z/i, " ASC") || (s << " DESC")
```

Comment:
Is this necessary change?

Reply:
Yeah; `s` can be an SqlLiteral, which implements many methods (including [I assume?] `concat`) for SQL construction. It's not great, but that's more Arel's fault.

I do have some vague thoughts on rearranging this method, but that's an unrelated refactoring from this PR.

Reply:
I see. Thanks for the explanation.